packages/api/portfolioDrafts/application/createApplicationStep.ts

```diff
@@ -108,28 +134,64 @@ export function performDataValidationOnApplication(application: Application): Ar
       validationMessage: ValidationMessage.INVALID_APPLICATION_NAME,
     });
   }
-  const environmentErrors = application.environments
+  const environmentErrors: ApplicationStepValidationErrors[] = application.environments
     .map(performDataValidationOnEnvironment)
     .reduce((accumulator, validationErrors) => accumulator.concat(validationErrors), []);
+  const operatorErrors: ApplicationStepValidationErrors[] = application.operators
+    .map(performDataValidationOnOperator)
+    .reduce((accumulator, validationErrors) => accumulator.concat(validationErrors), []);
 
-  return errors.concat(environmentErrors);
+  return errors.concat(environmentErrors, operatorErrors);
 }
 
 /**
  * Performs data validation on the given Environment object
  * @param environment the object containing data to validate
  * @returns a collection of application validation errors
  */
-export function performDataValidationOnEnvironment(environment: Environment): Array<ApplicationValidationError> {
-  const errors = Array<ApplicationValidationError>();
+export function performDataValidationOnEnvironment(environment: Environment): Array<ApplicationStepValidationErrors> {
+  const errors = Array<ApplicationStepValidationErrors>();
   if (environment.name.length < 4 || environment.name.length > 100) {
     errors.push({
-      applicationName: environment.name,
+      environmentName: environment.name,
       invalidParameterName: "name",
       invalidParameterValue: environment.name,
       validationMessage: ValidationMessage.INVALID_ENVIRONMENT_NAME,
     });
   }
+  const operatorErrors: ApplicationStepValidationErrors[] = environment.operators
+    .map(performDataValidationOnOperator)
+    .reduce((accumulator, validationErrors) => accumulator.concat(validationErrors), []);
+  return errors.concat(operatorErrors);
+}
+
+/**
+ * Performs data validation on the given Operator object
+ * @param operator the object containing data to validate
+ * @returns a collection of operator validation errors
+ */
+export function performDataValidationOnOperator(operator: Operator): Array<ApplicationStepValidationErrors> {
+  const errors = Array<ApplicationStepValidationErrors>();
+  if (operator.display_name.length < 1 || operator.display_name.length > 100) {
+    errors.push({
+      operatorDisplayName: operator.display_name,
+      invalidParameterName: "display_name",
+      invalidParameterValue: operator.display_name,
+      validationMessage: ValidationMessage.INVALID_OPERATOR_NAME,
+    });
+  }
+  // Using API Gateway "email" format to enforce the most common email rules
+  // and only ensuring the .mil top-level domain here.
+  // TODO: possibly add patterns into the internal API spec to help
+  const emailRegex = /.mil$/i;
+  if (!emailRegex.test(operator.email)) {
```

Comment:
Was going to ask why not `match` here, but after reading MDN, `test` seems more appropriate.